apps/explorer/src/pages/transaction-result/TransactionResult.tsx

```diff
@@ -1,19 +1,47 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import {
```

Comment:
We're eventually going to deprecate these getters, so can you just do the property access instead?